web_src/js/features/common-global.js

```diff
@@ -192,7 +192,8 @@ export function initGlobalDropzone() {
       thumbnailWidth: 480,
       thumbnailHeight: 480,
       init() {
-        this.on('success', (_file, data) => {
+        this.on('success', (file, data) => {
+          file.uuid = data.uuid;
```

Comment:
To be more future-proof, we could also do the following:
```suggestion
          file.uuid = data.uuid || file.uuid; // Replace the local UUID with the server one if present
```

Reply:
See my comment, no need to read the `file.uuid` again, and even if the `file.uuid` is read, there would be a more strange bug in the future if the `file` has an unrelated `uuid` builtin.

The code was quite dirty already, as a quick fix this PR is very suitable. To make the logic stable and clear, the file uploading related code should be all rewritten.